tests/test_parser.py

```diff
@@ -280,10 +282,9 @@ def test_flush (self, _in, _out):
 
     def test_entities (self):
         # Test entity resolving.
-        resolve = linkcheck.HtmlParser.resolve_entities
         for c in "abcdefghijklmnopqrstuvwxyz":
-            self.assertEqual(resolve("&#%d;" % ord(c)), c)
-        self.assertEqual(resolve("&#1114112;"), u"")
+            self.assertEqual(html.unescape("&#%d;" % ord(c)), c)
+        self.assertEqual(html.unescape("&#1114112;"), b"\xef\xbf\xbd".decode())
```

Comment:
I don't think it makes sense for linkchecker to have unit tests for non-linkchecker code.  We could just remove this test, since the code it was testing was removed.

(html is part of the standard library in Python 3.x, and I assume it comes from the future package on Python 2.)

Reply:
test_entities() removed.